lib/matplotlib/rcsetup.py

```diff
@@ -145,6 +145,13 @@ def validate_bool_maybe_none(b):
         raise ValueError('Could not convert "%s" to boolean' % b)
 
 
+def deprecate_axes_hold(value):
```

Comment:
Can you make this function private? It's unlikely that anyone is every going to use it, but who knows…